src/components/dashboard/insight-card/InsightCard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="insight-card">
+    <div class="main-data">
+      <span class="summary-value txt-2 bold">{{ mainData }}</span>
+      <span v-if="withStar" class="star txt-2">&#9733;</span>
+    </div>
+    <div class="main-description">
+      <p class="text txt-5">{{ mainDescription }}</p>
+    </div>
+    <div
+      v-if="additionnalDescription"
+      class="additionnal-description"
+      >
+      <p class="subtext txt-5 semibold">{{ additionnalDescription }}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'insight-card',
+  props: {
+    mainData: { type: String, required: true },
+    mainDescription: { type: String, required: true },
+    additionnalDescription: { type: String },
+    withStar: { type: Boolean, default: false }
+  },
+  methods: {
+    // subtextModifier(){
```

Comment:
Can we not commit commented out code? This does not add any value at all and just makes it harder to maintain.

in some weeks, you might need to do a change to do this component, and by then you won't be sure why it's commented out yet. (Should this be commented out? Should this still be implemented? Is there a bug in that piece of code? Can I safely uncomment it?)

Most of the times, it's just easier to remove it (it' still in git if you want to find it back) and to rewrite it once you actually need it.

Reply:
Yes true, It is an oversight, well spotted!